app/views/content/what-pension-does-a-teacher-get.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "What pension does a teacher get?"
+heading: "What pension does a teacher get?"
+description: |-
+  The teachers' pension is one of the most generous in the country. Learn more about the benefits of the pension scheme once you become a teacher.
+date: "2022-11-29"
+image: false
+promo_content:
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-salaries
+related_content:
+    Salaries and benefits : "/salaries-and-benefits"
+    Steps to become a teacher : "/steps-to-become-a-teacher"
+    Train to be a teacher if you have or are studying for a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Scholarships and bursaries for training to teach : "/funding-and-support/scholarships-and-bursaries"
+keywords:
```

Comment:
@EllieNodder more of a question for you re frontmatter - should we keep keywords just to relevant pension content?

Reply:
I _think_ so. If someone's looking for 'how to become a teacher' or 'teacher training courses', this isn't really the content they'd expect to be given.

Reply:
Stuffing is the best part of a Christmas dinner.